src/app/donation-complete/donation-complete.component.ts

```diff
@@ -98,6 +98,13 @@ export class DonationCompleteComponent {
   }
 
   loginCaptchaReturn(captchaResponse: string) {
+    if (captchaResponse === null) {
+      // This is expected after ~1 min when the code expires. At this point we should
+      // never be executing the login again because if the captcha was set up at all then
```

Comment:
Same as https://github.com/thebiggive/donate-frontend/pull/689/files#r994633304

Reply:
Same explanation: there is no local var and also no case where the captcha should execute a 2nd time in this component.